src/wiki/plugins/images/models.py

```diff
@@ -119,13 +119,24 @@ def on_image_revision_delete(instance, *args, **kwargs):
     if not instance.image:
         return
 
-    # Remove image file
-    instance.image.delete(save=False)
-
+    path = None
     try:
         path = instance.image.path.split("/")[:-1]
+        path = os.path.dirname(instance.image.path)
```

Comment:
Hi @Steckelfisch -- path seems to be assiged twice here. I think the line above should be deleted?